docs/wiki/frameworkx/fortify-framework.md

```diff
@@ -69,9 +69,9 @@ class ACEX_Fortify_Presets {
 
 A custom deploy handler allows missions makers to decide if an object can be placed or not.
 
-To verify that an object isn't above a certain terrain height we can check the heigt of the object before it is confirmed as placed. Returning `false` from the code block means that placement is not allowed.
+To verify that an object isn't above a certain terrain height we can check the height of the object before it is confirmed as placed. Returning `false` from the code block means that placement is not allowed.
 
-```sqf
+```cpp
```

Comment:
This should still be SQF or is the wiki not supporting the syntax?

Reply:
It is not. Check how it looks now (brokeb).

Reply:
github syntax highlighting != gh pages' rouge syntax highlighting